app/router.js

```diff
@@ -7,9 +7,9 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('reminders');
+  this.route('reminders', function() {
+    this.route('reminder', {path: '/reminder/:reminder_id'});
```

Comment:
Because we're nested within the `'/reminders'` route it will automatically prepend the path with `reminders`. this line should simply read `'reminder',  {path: '/:reminder_id}` because otherwise you will see `reminders/reminder/1` in the URL which is a bit redundant.